docs/features/qualityofservice.rst

```diff
@@ -35,13 +35,18 @@ Then add the following section to a Route configuration:
   "QoSOptions": {
     "ExceptionsAllowedBeforeBreaking": 3,
     "DurationOfBreak": 1000,
-    "TimeoutValue": 5000
+    "TimeoutValue": 5000,
+	"FailureRatio": .8 // .8 = 80% of failed, this is default value
+	"SamplingDuration": 10000 // The time period over which the failure-success ratio is calculated (in milliseconds), default is 10000 (10s)
```

Comment:
1. Better to add new properties docs below, after line 45.
2. You are wrong regarding default value! According to the docs for [SamplingDuration](https://www.pollydocs.org/api/Polly.CircuitBreaker.CircuitBreakerStrategyOptions-1.html#Polly_CircuitBreaker_CircuitBreakerStrategyOptions_1_SamplingDuration), the default value is 30 seconds! So, we override library value by our custom one (10s). I believe we must assign lib value in `QoSOptions`